jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -441,13 +524,30 @@ def _update_viewer_filters(self, event={}):
             # only want spectral viewers in the options
             self.add_results.viewer.filters = ['is_spectrum_viewer']
 
-    def vue_apply(self, event):
+    def calculate_fit(self, add_data=True):
```

Comment:
it might be nice to have this also take an API override for `results_label`, but that should probably be done across the board for all plugins at once and with shared code.